### ISSUE TITLE
Create a DockerHub organisation for AS

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#1322]
I'm creating this issue to document what was done, for prosperity.

Note from Docker's reply approval email:

>If you haven’t already, please take the time to update your project’s Hub pages to include a detailed project description, links to your project source code, as well as contributing guidelines, and a link to your organization’s website. Projects lacking this information may not receive the Docker Sponsored Open Source badging for their images on Docker Hub.

Will leave this issue open until we have added this, to make sure the next year we get an extension of this plan.

**Comment by kinow** [comment]
Today I created an account `autosubmit`. Then I went into settings, convert, and converted it into organisation. In order to do that, I had to assing my user `kinow` as owner.

Then, once I had the organisation, I applied for the Docker Open Source program, which improves the rate limits for downloads. I tried to be very clear that it is a research tool, created +10 years ago, used in different sites. For the field for Sponsorship, I explained it's mainly workers from the BSC, but with contributions of other institutions, that pay their employers to use/contribute to Autosubmit.

I submitted the request, and it was approved in about 1 hour.

**Comment by kinow** [comment]
Their email:

Hi Bruno Kinoshita,

Congratulations! Your project has been approved for the Docker-Sponsored Open Source program. A Docker Team subscription will be allocated to the project organization Docker ID specified in your application within the next 3 weeks. In addition to a Team subscription, you now have:

* free autobuilds
* rate-limit removal for all users pulling public images from your project namespace
* special badging on Docker Hub (this will be visible within two weeks)

If you haven’t already, please take the time to update your project’s Hub pages to include a detailed project description, links to your project source code, as well as contributing guidelines, and a link to your organization’s website. Projects lacking this information may not receive the Docker Sponsored Open Source badging for their images on Docker Hub.

Membership in the Docker-Sponsored Open Source (DSOS) program is valid for one year. You will receive a reminder email from us when it is time to renew your membership.

We are thrilled to have you as an open-source partner. If you have any questions regarding the open source program please contact opensource@docker.com and include the name of your project in the subject line. If you have questions regarding technical support please contact support@docker.com.

Thank you!

**Comment by kinow** [comment]
I also had to create a repository, for the application to the open source program. I created `autosubmit` repository, empty, and submitted the application.

After that, I created a read+write token, and uploaded the image for 4.1.7 (after patching with fixes by @lteckent in !440 - thanks).

@dbeltrankyl is a owner of the org too, now.

**Comment by kinow** [comment]
Done. I used the PyPI and Git descriptions for the text, and the Autosubmit docker/README.md file as base for the overview of the Autosubmit repo. For the other repositories I added some text explaining that those are for EDITO.eu, and in the case of autosubmit/gui that's being updated for general use (we can update it at any time, what's important is to have the data so we don't lose the open source status).

Closing!
